app/src/lib/components/HunkLines.svelte

```diff
@@ -34,42 +39,51 @@
 		});
 		return tokens;
 	}
-
-	$: isSelected = selectable && selected;
 </script>
 
-<div class="code-line" role="group" style="--tab-size: {tabSize}" on:contextmenu|preventDefault>
-	<div class="code-line__numbers-line">
-		<button
-			on:click={() => selectable && dispatch('selected', !selected)}
-			class="numbers-line-count"
-			class:selected={isSelected}
-			style:min-width={minWidth + 'rem'}
-			style:cursor={draggingDisabled ? 'default' : 'grab'}
-		>
-			{line.beforeLineNumber || ''}
-		</button>
-		<button
-			on:click={() => selectable && dispatch('selected', !selected)}
-			class="numbers-line-count"
-			class:selected={isSelected}
-			style:min-width={minWidth + 'rem'}
-			style:cursor={draggingDisabled ? 'default' : 'grab'}
+<div
+	class="line-wrapper"
+	style="--tab-size: {tabSize}; --minwidth: {minWidth}rem; --cursor: {draggingDisabled
+		? 'default'
+		: 'grab'}"
```

Comment:
Hmm, minWidth should probably be expressed in px now.. but I'm wondering if inline styles will be translated? @PavelLaptev

Reply:
So based on our conversation, for inline styles they're either supposed to be expressed in `px` and use the helper fn or just directly in `rem`'s, right?

Reply:
[pxToRem](https://github.com/cuth/postcss-pxtorem#readme) plugin can't convert `PX` that were written inline in the style attribute. In this case, if it's not a component property, you can use `REM` directly. Or if it's a property you can use a helper function like in this file https://github.com/gitbutlerapp/gitbutler/blob/cdd596f8bc4b9801671793360184f1236766f0c6/app/src/lib/components/Spacer.svelte#L11